app/models/course.rb

```diff
@@ -57,6 +57,10 @@ class Course < ApplicationRecord
 
   DEGREE_REQUIREMENTS_REQUIRED_FROM = 2022
 
+  # Most providers require GCSE grade 4 ("C"),
+  # but some require grade 5 ("strong C")
+  PROVIDERS_REQUIRING_GCSE_GRADE_5 = %w[U80 I30].freeze
```

Comment:
Will we need to consider this to be in the database at a later point in time?

Or are they just the only 2 out of 1100?

Reply:
I suppose this is pragmatic while we only have 2 🤷